pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_submitting-documentation-feedback.adoc

```diff
@@ -0,0 +1,19 @@
+_module-type: PROCEDURE
+[id="proc_submitting-documentation-feedback_{context}"]
+= Submitting documentation feedback
+
+[role="_abstract"]
+If you encounter any documentation bugs or have any requests for additions or changes to the {ProductName} end-user documentation, you can open a Jira ticket to create a request for the documentation team.
+
+Do not submit documentation feedback to submit a bug or functionality request for Pantheon.
+
+.Procedure
+
+. Log in to link:https://projects.engineering.redhat.com/projects/FCCEUD/issues[the Jira project for the Pantheon 2 end-user documentation (FCCEUD)]. This is the internal Red Hat instance of Jira, so you need to be connected to the Red Hat network in an office or using the VPN.
+. Click *Create*.
+. Fill out the form as follows:
++
```

Comment:
You can keep it as it is or use substeps. The mod-docs templates use substeps

Reply:
@dvagnero The type of list to use should be guided by the type of info. Steps or substeps indicate a necessary order to follow. In this case, we're just describing a set of fields to fill out. It doesn't really matter what order, and I think that a two column list (i.e. definition list) is more semantically correct and easier to parse visually.

There are a lot of Asciidoc features that the templates don't include.